src/browser/provider/index.js

```diff
@@ -135,7 +135,16 @@ export default class BrowserProvider {
         return width <= maxScreenSize.width && height <= maxScreenSize.height;
     }
 
-    async _maximizeLocalBrowserWindow (browserId) {
+    async _maximizeLocalBrowserWindow (browserId, currentWidth, currentHeight) {
+        // NOTE: no true maximizing on macOS: https://github.com/DevExpress/testcafe-browser-tools/issues/89
+        if (OS.mac) {
```

Comment:
Since browserTools.maximize don't work on Mac now I think we should move this `maximize -> resizeWindow` fallback to the browserTools logic